### PR TITLE
Decode TRAJ + some final polish

### PR DESCRIPTION
**Pull request by LukasWikander**
Needed to redo a bit of the decodeMsg methods since they assumed the ISO header had been stripped from the data already (which does not work with ISO lib decode functions)

### CODE REVIEW

File: Linux/Car_Client/chronoscomm.h [hunk 1]
```diff
@@ -172,7 +169,8 @@ typedef struct {
 
 // SYNC Word
 #define ISO_SYNC_WORD                   0x7E7E
```
Comment (konglobemeralt) [inline Linux/Car_Client/chronoscomm.h]: Might want to remove this while we are at it, it is replaced by 7E7F right?
```suggestion

```

**Review by konglobemeralt — APPROVED** [review]
chronoscomm lives to fight another day!

File: Linux/Car_Client/chronoscomm.cpp [hunk 2]
```diff
@@ -218,7 +220,7 @@ void ChronosComm::sendTraj(chronos_traj traj)
     for (chronos_traj_pt pt: points) {
         vb.vbAppendUint16(ISO_VALUE_ID_REL_TIME);
         vb.vbAppendUint16(4);
-        vb.vbAppendUint32(pt.tRel); // TODO: Multiply with 4?
+        vb.vbAppendUint32(pt.tRel.count()); // TODO: Multiply with 4?
```
Comment (konglobemeralt) [inline Linux/Car_Client/chronoscomm.cpp]:
```suggestion
        vb.vbAppendUint32(pt.tRel.count()); 
```

This TODO was added by Vedder four years ago. I think it is safe to say we will not multiply by 4.

File: Linux/Car_Client/chronoscomm.cpp [hunk 3]
```diff
@@ -617,8 +658,8 @@ void ChronosComm::readPendingDatagrams()
 
         vb.remove(vb.size() - 2, 2);
 
-        if (true || checksum == 0) {
-            decodeMsg(message_id, message_len, vb, sender_id);
+        if (true /* || checksum == 0 */) {
```
Comment (samuelthoren) [inline Linux/Car_Client/chronoscomm.cpp:661]: Is this temporary, or could we remove the if-statement?

**Review by samuelthoren — APPROVED** [review]
Looks good! There are some stuff that you have commented out, should those be removed?

File: Linux/Car_Client/chronoscomm.cpp (hunk 3 above)
Comment (LukasWikander) [reply Linux/Car_Client/chronoscomm.cpp:661]: It's mostly as a reminder to check the checksum when that calculation has been implemented.

**Pull request merged**